cmd/terramate/cli/cloud.go

```diff
@@ -42,25 +64,168 @@ func credentialPrecedence(output out.O, clicfg cliconfig.Config) []credential {
 	}
 }
 
-func (c *cli) cloudInfo() {
-	cred, err := c.loadCredential()
+func (c *cli) checkSyncDeployment() {
+	if !c.parsedArgs.Run.CloudSyncDeployment {
+		return
+	}
+	err := c.setupSyncDeployment()
 	if err != nil {
+		if errors.IsKind(err, ErrOnboardingIncomplete) {
+			c.cred().Info()
+		}
 		fatal(err)
 	}
 
-	cloud := cloudConfig{
-		baseAPI: cloudBaseURL,
-		client:  &http.Client{},
-		output:  c.output,
+	if orgs := c.cred().organizations(); len(orgs) != 1 {
+		fatal(
+			errors.E("requires 1 organization associated with the credential but %d found: %s",
+				len(orgs),
+				orgs),
+		)
+	}
+
+	c.cloud.run.meta2id = make(map[string]int)
+
+	if runid := os.Getenv("GITHUB_RUN_ID"); runid != "" {
```

Comment:
Run ID is not an uuid and should not be used for the deployment as it does not change between runs in GitHub..

Reply:
yes true. I was using this env to force a fixed deployment ID when using the fake server. I guessed this GH env was an UUID but yeah, just removed as it's out of the scope now.